### DISCUSSION TITLE
diary/2020/08/02/how-to-study-business/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#158] [Comments]
# diary/2020/08/02/how-to-study-business/

 비즈니스를 어떻게 공부해야 하는지에 대한 개인 경험을 담은 글입니다 이 글이 진리라고 생각하진 않고, 제가 어떻게 공부하고 고민했는지에 대한 내용이 주로 담겨있습니다 피드백은 언제나 환영합니다! 

https://zzsza.github.io/diary/2020/08/02/how-to-study-business/

**Comment by xpmxf4** [comment]
요새 단순히 특정 지표, 수치보다 좀 더 심화된 무언가 없나를 찾다 이 글을 찾게 됐네요!!
지금 제 고민에 되게 와닿는 글이라서,  “한 장으로 끝내는 비즈니스 모델 100” 책까지 바로 주문했습니다!!

이번 10기 글또에서 길벗 출판사 리뷰 이벤트에 당첨된 책 이후에, 위 책에 대한 글을 읽고 
제 회사의 비즈니스를 좀 더 파악해보고 이에 대한 도표를 만들어 회사에 아얘 문서화 겸 이에 대한 글을 작성해볼 플랜이 생각났습니다!! 

좋은 글 너무 감사합니다!!
